### ISSUE TITLE
mirage configure inserts camlp4 into Makefile

### ISSUE THREAD

**Issue by sgrove** [mirage/mirage#502]
Seems like this isn't necessary (and caused some problems with my jsoo build chain), and can be removed.
```
SYNTAX = -tags "syntax(camlp4o),annot,bin_annot,strict_sequence,principal,thread"

SYNTAX += -tag-line "<static*.*>: -syntax(camlp4o)"
```

**Comment by samoht** [comment]
That's clearly should be fixed. A PR is more than welcome!

**Comment by samoht** [comment]
Related to #405

**Comment by samoht** [comment]
Fixed by #510
